### DISCUSSION TITLE
Megahit assembly failing after 16 hours

### DISCUSSION THREAD

**Discussion by rachel-1212** [nextflow-io/nextflow#5491] [Q&A]
Hi, I'm new to using Nextflow therefore wondering if someone could help with me this. I am running the Metatdenovo pipeline via a remote computer with conda. I managed to run the pipeline successfully when I tested it with one sample, however when I try to run for my full set of 16 samples, it fails at the megahit_interleaved step. There is no exit code, it just seems to stop after 16 hours every time I run it. I have tried increasing the cpus for this step using a config file, but that hasn't helped. Any suggestions would be much appreciated.

**Comment by bentsherman** [comment]
Try asking on the pipeline repo, or the [community forum](https://community.seqera.io/), maybe they can help
